packages/invoices/src/OutboundShipment/DetailView/modals/hooks/useDraftOutboundLines.tsx

```diff
@@ -52,7 +52,7 @@ const createDraftOutboundLine = ({
   invoiceId,
   stockLineId: stockLine.id,
   location: invoiceLine?.location,
-  expiryDate: invoiceLine?.expiryDate,
+  expiryDate: invoiceLine?.expiryDate || stockLine?.expiryDate,
```

Comment:
`expiryDate` was only being shown for invoice lines.. which wasn't very helpful 😉